ui/tests/integration/components/toolbar-actions-test.js

```diff
@@ -0,0 +1,16 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { isPresent } from 'ember-cli-page-object';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | toolbar-actions', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    await render(hbs`<ToolbarActions>These are the toolbar actions</ToolbarActions>`);
+
+    assert.equal(this.element.textContent.trim(), 'These are the toolbar actions');
+    assert.ok(isPresent('.toolbar-actions'));
```

Comment:
We're using `qunit-dom` for simple tests / assertions like this - and you can nix the extra import that way too.
```suggestion
    assert.dom('.toolbar-actions').exists();
```